src/plugins/intel_cpu/src/emitters/snippets/x64/kernel_executors/brgemm_copy_b.cpp

```diff
@@ -295,6 +295,12 @@ void BrgemmCopyBKernel::emit_brgemm_copy_b_kernel_call(size_t N,
     spill.postamble();
 }
 
+std::set<snippets::Reg> BrgemmCopyBKernel::get_live_regs() const {
+    return {{snippets::RegType::gpr, static_cast<size_t>(src_reg.getIdx())},
+            {snippets::RegType::gpr, static_cast<size_t>(tr_src_reg.getIdx())},
+            {snippets::RegType::gpr, static_cast<size_t>(comp_reg.getIdx())}};
```

Comment:
It looks like it makes sense to save these registers only if the callee doesn't. Am I right?
We can easily do it by checking that the regs are not in `get_callee_saved_reg_idxs()`.

Reply:
I thought about that. But we know that all these registers are ABI registers. They're not callee-saved. So I decided just to return this set without any additional extra checks.

Reply:
Oh, I see. Could you add a comment please, so we wouldn't be confused in the future?

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/28635/commits/13df9f6fcd95e907507e181721629554ca93c1ee